lib/task-data/schemas/dell-wsman-get-config-catalog.json

```diff
@@ -0,0 +1,81 @@
+{
+    "copyright": "Copyright 2017 Dell Inc. or its subsidiaries. All Rights Reserved.",
+    "definitions": {
+        "serverIP": {
+            "description": "Dell server iDARC IP address",
+            "type": "string"
+        },
+        "serverUsername": {
+            "description": "Dell server iDARC username",
+            "type": "string"
+        },
+        "serverPassword": {
+            "description": "Dell server iDARC password",
+            "type": "string"
+        },
+        "shareAddress": {
+            "description": "NFS/CIFS server address which holds firmware repository",
+            "type": "string"
+        },
+        "shareUsername": {
+            "description": "NFS/CIFS server usename",
+            "type": "string"
+        },
+        "sharePassword": {
+            "description": "NFS/CIFS server password",
+            "type": "string"
+        },
+        "shareName": {
+            "description": "Share path of the firmware repository folder on share server",
+            "type": "string"
+        },
+        "shareType": {
+            "description": "Specify whehter use NFS or CIFS as share server to hold firmware repository",
+            "type": "integer",
+            "enum": [0, 2]
```

Comment:
I think you should explanation what the number `0` and `2` means. For to be better, you could use a string  type, user directly inputs `nfs` or `cifs`, so that user doesn't have to remember the mapping.

Reply:
I thought about change to nfs cifs as my previous firmware update PR. But this will bring in API change in this Graph. The original get component task is using 0/2 as input, and there are some other teams already using this API. So I have to stay the same

Reply:
Actually, you could support both, and mark the number format as `deprecated`.  So you don't break the compatibility but also encourage user to use the better one.